pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -278,6 +283,16 @@ func (o *StartBuildOptions) Complete(f *clientcmd.Factory, in io.Reader, out, er
 	}
 	o.EnvVar = env
 
+	// Handle Docker build arguments. In order to leverage existing logic, we
+	// first create an EnvVar array, then convert it to []docker.BuildArg
+	cmdutil.WarnAboutCommaSeparation(o.ErrOut, o.Args, "--build-args")
```

Comment:
we don't need to warn about comma separate for this flag because it never supported comma separation before.